flan/src/lib/MessageContents.svelte

```diff
@@ -1,23 +1,50 @@
 <script lang="ts">
 	import { MessageType, type IMessage } from './types';
 	import FileIcon from './svgs/File.svelte';
-	import { getModalStore, type ModalSettings } from '@skeletonlabs/skeleton';
+	import { getModalStore, getToastStore, type ModalSettings, type ToastSettings } from '@skeletonlabs/skeleton';
 
+	const toastStore = getToastStore();
 	const modalStore = getModalStore();
 	export let message: IMessage;
 	$: isLink = message.text?.startsWith('http');
+	$: isPassword = message.text?.includes('||');
+	$: messageSplit = isPassword && message.text?.split(' ');
 	$: modal = {
 		type: 'component',
 		component: 'imageModal',
-		meta: {image: message.payload?.src, name: message.payload?.name}
+		meta: { image: message.payload?.src, name: message.payload?.name }
 	} as ModalSettings;
+	function copySpoiler(e: MouseEvent) {
+		if (!e) return;
+		const wordElement = e.target as HTMLElement
+		navigator.clipboard.writeText(wordElement.innerText);
+		const toastMessage: ToastSettings = {
+			message: 'Copied to clipboard 🤓',
+			background: 'variant-filled-success'
+		};
+		toastStore.trigger(toastMessage);
+	}
 </script>
 
 {#if message.type === MessageType.Text}
 	{#if isLink}
 		<a href={message.text} target="_blank" class="break-all text-secondary-300">
 			{message.text}
 		</a>
+	{:else if isPassword && messageSplit}
```

Comment:
The && here does nothing as if there is no spaces in the string, it will evaluate to ['']
![image](https://github.com/ProPablo/nakl/assets/40000658/db3b9421-02a8-4446-bbf2-9b5854825922)